osu.Game/OsuGame.cs

```diff
@@ -1023,9 +1010,14 @@ protected override void UpdateAfterChildren()
         {
             base.UpdateAfterChildren();
 
-            screenOffsetContainer.Padding = new MarginPadding { Top = ToolbarOffset };
+            ScreenOffsetContainer.Padding = new MarginPadding { Top = ToolbarOffset };
             overlayContent.Padding = new MarginPadding { Top = ToolbarOffset };
 
+            ScreenOffsetContainer.X = 0f;
```

Comment:
This line is causing every-frame invalidations when either of the overlays are visible.